src/virtualenv/activation/bash/activate.sh

```diff
@@ -46,7 +46,12 @@ deactivate () {
 # unset irrelevant variables
 deactivate nondestructive
 
-VIRTUAL_ENV="$(if [ "$OSTYPE" = "cygwin" ]; then cygpath -u '__VIRTUAL_ENV__'; else echo '__VIRTUAL_ENV__'; fi;)"
+VIRTUAL_ENV='__VIRTUAL_ENV__'
+
+if [ "$OSTYPE" = "cygwin" ]; then
+    VIRTUAL_ENV=$(cygpath -u "$VIRTUAL_ENV")
```

Comment:
we should validate that cygpath is an existing command, and ignore if it fails 🤔

Reply:
I've tested the updated code on cygwin & msys, and it works for both.

I've added the check to make sure cygpath exists before we invoke it. Do you mean that we should also ignore the error if invoking cygpath fails? I think we probably want to fail the operation in that case, don't we, since the virtualenv isn't going to work?